src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -114,4 +114,6 @@
   @Column(columnDefinition = "timestamp with time zone")
   @UpdateTimestamp
   private OffsetDateTime lastUpdated;
+
+  @Column private boolean handDelivery;
```

Comment:
Needs `@Column(nullable = false, columnDefinition = "BOOLEAN DEFAULT false")` to match the DDL